Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -24,6 +24,11 @@ namespace gdjs {
       isCollidingAnyPlatform: false,
     };
 
+    /**
+     * A very small value compare to 1 pixel, yet very huge compare to rounding errors.
+     */
+    private static readonly epsilon = 2 ** -20;
```

Comment:
Might be worth using 1 / (2 << xxx) for consistency with the other epsilons?

Reply:
They are only in tests. I didn't know this syntax, I will change them next time I add a test.